language/golang/build-images.md

```diff
@@ -0,0 +1,388 @@
+---
+title: "Build your Go image"
+keywords: containers, images, go, golang, dockerfiles, coding, build, push, run
+description: Learn how to build your first Docker image by writing a Dockerfile
+redirect_from:
+- /get-started/golang/build-images/
+---
+
+{% include_relative nav.html selected="1" %}
+
+## Prerequisites
+
+* Some understanding of Go and its toolchain. This is not a tutorial on Go. If you are new to the language, the [Go website](https://golang.org/){: target="_blank" rel="noopener" class="_"} is a good starting point, so go (pun intended) check it out.
+* Some awareness of basic Docker concepts. If unsure, work through the orientation and setup in Get started [Part 1](/get-started/).
+
+## Overview
+
+Now that we have a good overview of containers and the Docker platform, let’s take a look at building our first image. An image includes everything you need to run an application - the code or binary, runtime, dependencies, and any other file system objects required.
+
+To complete this tutorial, you need the following:
+
+- Go version 1.16 or later. 
+- Docker running locally. Follow the instructions to [download and install Docker](https://docs.docker.com/desktop/).
+- An IDE or a text editor to edit files. We recommend using [Visual Studio Code](https://code.visualstudio.com/){: target="_blank" rel="noopener" class="_"}.
+
+## Introducting the sample application
+
+To avoid losing focus on Docker's features, the sample application is a minimal HTTP server that has only three features:
+
+* It responds with a text message containing a heart symbol ("<3") on requests to `/`.
+* It responds with `{"Status" : "OK"}` to the health check request on requests to `/ping`.
+* The port it listens on is configurable using the environment variable `HTTP_PORT`. The default value is `8080`.
+
+Thus, it somewhat mimics enough basic properties of a REST microservice to be useful for our learning of Docker.
+
+The source code for the application is in the [olliefr/docker-gs-ping](https://github.com/olliefr/docker-gs-ping){: target="_blank" rel="noopener" class="_"} GitHub repo. Please feel free to clone or fork it.
+
+For our present study, we clone it to our local machine:
+
+```shell
+$ git clone github.com/olliefr/docker-gs-ping
```

Comment:
Add `https://` to make `git clone` work.

```suggestion
$ git clone https://github.com/olliefr/docker-gs-ping
```

Reply:
Interesting point! Fixed.